### PR TITLE
Official Documents Page - use hosted documents

### PR DESCRIPTION
**Pull request by briaguya**
resolves https://github.com/cityofaustin/techstack/issues/3057
resolves https://github.com/cityofaustin/techstack/issues/2972

### CODE REVIEW

File: joplin/db/system-generated/seeding.datadump.json [hunk 1]
```diff
@@ -9331,8 +9211,8 @@
 {
   "model": "users.user",
   "fields": {
-    "password": "pbkdf2_sha256$150000$wg2Rdshfdqly$3VTEDmmvBrEigN93vPEgx/dMD8675aHOMAtCGPv/vWw=",
-    "last_login": "2019-09-27T05:10:50.496Z",
+    "password": "pbkdf2_sha256$100000$9fpvtivuWOYg$rOB3vDHoYJyEYs35zNdIJ0BmHvR8n5daAVwz7HoeC2Q=",
```
Comment (Niicck) [inline joplin/db/system-generated/seeding.datadump.json]: Did this password not get sanitized? Or was this added locally?

File: scripts/migration-test.sh [hunk 2]
```diff
@@ -138,7 +138,9 @@ function handle_input {
     echo "#### Glad that worked. We'll create a new migration_datadump for you."
 
     # Build new migration datadump
-    docker exec -it ${COMPOSE_PROJECT_NAME}_app_1 python joplin/manage.py dumpdata --indent 2 --natural-foreign --natural-primary | \
+    # Excluding wagtailcore.GroupCollectionPermission because of IntegrityError issues
+    # see: https://docs.djangoproject.com/en/2.2/topics/serialization/#natural-keys for details
+    docker exec -it ${COMPOSE_PROJECT_NAME}_app_1 python joplin/manage.py dumpdata --exclude=wagtailcore.GroupCollectionPermission --indent 2 --natural-foreign --natural-primary | \
```
Comment (Niicck) [inline scripts/migration-test.sh]: Tell me about `--exclude=wagtailcore.GroupCollectionPermission`. That fixed the document dump issue?

File: joplin/db/system-generated/seeding.datadump.json (hunk 1 above)
Comment (briaguya) [reply joplin/db/system-generated/seeding.datadump.json]: I honestly am not sure what's going on with it, but I've been able to log in as multiple users with sanitized pws so there's that at least

File: scripts/migration-test.sh (hunk 2 above)
Comment (briaguya) [reply scripts/migration-test.sh]: It really is still a mystery to me. Quite a bit of the dump/load data stuff still feels like magic to me.

**Pull request merged**